app/enums/TransactionName/TransactionName.scala

```diff
@@ -116,4 +116,8 @@ case object RefundToTaxPayer extends TransactionName {
 
 case object CreditsSummary extends TransactionName {
   val name = "credits-summary"
+}
+
+case object ManageIncomeSourcesHome extends TransactionName {
+  val name = "view-and-manage-income-sources-home"
```

Comment:
in the story we have auditType set as "ViewAndManageIncomeSourcesHome", 
so we need to update either story or code to be consistent;